### DISCUSSION TITLE
[꿀팁] 이미지 업로드

### DISCUSSION THREAD

**Discussion by karim-moon** [bcaitech1/p3-dst-chatting-day#8] [Tips]
![스크린샷 2021-04-28 오후 4 36 52](https://user-images.githubusercontent.com/52662993/117245405-c84f9f00-ae75-11eb-941f-55effcf256c1.png)

디스커션에 사진 넣을 때 사진 끌어다 쓰시면됩니다!

**Comment by changwoomon** [comment]
<img src="https://user-images.githubusercontent.com/59998179/117245539-ff25b500-ae75-11eb-9dba-a06a51376064.jpg" width=400>
